src/main/java/ir/mimlang/jmim/lang/std/filestream/JsonToMap.kt

```diff
@@ -0,0 +1,30 @@
+package ir.mimlang.jmim.lang.std.filestream
```

Comment:
Well, having such library is a good idea, but I have to say that what you wrote here is not mature enough to be valuable at meantime.

For instance, the package name doesn't match what it does in action, or treating all json strings as a `JSONObject` is no appropriate; you may use either `JSONObject` or `JSONArray`.

Reply:
I encourage you to reserve this for future use as a **_jar-plug-in_** to the language, which I need to provide an api for this purpose.